apps/etterlatte-vedtaksvurdering/src/main/kotlin/vedtaksvurdering/VedtaksvurderingRepository.kt

```diff
@@ -47,22 +47,22 @@ class VedtaksvurderingRepository(datasource: DataSource) {
             "vedtakstatus" to VedtakStatus.BEREGNET.name,
             "datovirkfom" to Date.valueOf(virkningsDato),
             "beregningsresultat" to beregningsresultat?.let { objectMapper.writeValueAsString(it) },
-            "vilkaarsresultat" to vilkaarsresultat.let { objectMapper.writeValueAsString(it) }
+            "vilkaarsresultat" to vilkaarsresultat?.let { objectMapper.writeValueAsString(it) }
         ),
         loggtekst = "Oppretter vedtak behandlingid: $behandlingsId sakid: $sakid"
     )
 
     fun oppdaterVedtak(
         behandlingsId: UUID,
         beregningsresultat: Beregningsresultat?,
-        vilkaarsresultat: VilkaarsvurderingDto,
+        vilkaarsresultat: VilkaarsvurderingDto?,
         virkningsDato: LocalDate
     ) = repositoryWrapper.oppdater(
         query = "UPDATE vedtak SET datovirkfom = :datovirkfom, beregningsresultat = :beregningsresultat, vilkaarsresultat = :vilkaarsresultat WHERE behandlingId = :behandlingid", // ktlint-disable max-line-length
         params = mapOf(
             "datovirkfom" to Date.valueOf(virkningsDato),
             "beregningsresultat" to objectMapper.writeValueAsString(beregningsresultat),
```

Comment:
Gjør du samme endring her? :D

Reply:
Ah, jeg tror jeg skjønner -- tilsvarende lagring for beregningsresultat også

Reply:
Jepp!